### DISCUSSION TITLE
Link as QR code. Scans not tracked?

### DISCUSSION THREAD

**Discussion by jaredcat** [YOURLS/YOURLS#3549] [Q&A] [answered]
I took a shortened url from my yourls instance and fed it though and online QR code generator ([QR code monkey](https://www.qrcode-monkey.com)) .
When I scan this, it usually doesn't seem to track the scan as a click.

Has anyone else ran into this? Is there a workaround to track "scans"?

**Comment by ozh** [comment, marked as answer]
If you mean : "is there a way to know when someone scans a QR code", the answer is no. You'll have a hit on this short URL only if they follow the link after they "scan" it.

**Reply by jaredcat** [reply]
> If you mean : "is there a way to know when someone scans a QR code", the answer is no. You'll have a hit on this short URL only if they follow the link after they "scan" it.

No, I mean even after following it doesn't seem to increase the amount of clicks on a shortened url

**Reply by ozh** [reply]
Following a short link always registers as a hit, unless the client has already stored the redirection in cache.
